omero/autogen_docs

```diff
@@ -86,6 +91,10 @@ done
 echo "Getting db properties"
 omero/autogen_db_version.py $WORKSPACE/OMERO.server > omero/conf_autogen.py
 
+echo "Update dependencies"
+bash omero/dependencies.sh
```

Comment:
The purpose of this script seems to be largely identical to `omero/autogen_db_version.py` above except it's in Bash vs Python and this is downloaded the latest release server vs using `OMERO.server`.
What was the rationale for not amending the existing logic?

Reply:
It is all a bit messy since we have bash script invoking python script. I opted for a bash script since the source scrip is a bash one